README.md

```diff
@@ -454,7 +454,7 @@ See the complete cmake file and how to link your own library [here](https://gith
 
 ## SDK Architecture
 
-At the heart of our SDK is, what we refer to as, [Azure Core](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core). This code defines several data types and functions for use by the client libraries that build on top of us such as an [Azure Storage Blob](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/storage) client library and [Azure IoT client libraries](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/iot). Here are some of the features that customers use directly:
+At the heart of our SDK is, what we refer to as, [Azure Core](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core). This code defines several data types and functions for use by the client libraries that build on top of us such as an [Azure Storage Blob](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/storage) client library and [Azure IoT client libraries](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot). Here are some of the features that customers use directly:

```

Comment:
Question: Why do we prefer `/blob/` over `/tree/`?

Reply:
Actually, github link is able to switch each other when necessary. I am using a script to do the replacement, so I just choose one of them as I feel like blob is used more often than tree. Just personal feeling

Reply:
In general blob should be used for files and tree should be used for directories but github usually forwards any blob requests that point to a directory to tree.